project-1_test/main.py

```diff
@@ -0,0 +1,144 @@
+import inspect
+import logging
+import types
+
+import gspread
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.chrome.service import Service as ChromeService
+from webdriver_manager.chrome import ChromeDriverManager
+from selenium.common.exceptions import NoSuchElementException
+import os, json
+from dotenv import load_dotenv
+load_dotenv()
+KEY = os.getenv("PROJECT_1202")
+option = webdriver.ChromeOptions()
```

Comment:
You can create a function to setup your chromedriver

Reply:
I have done that.